compiler/rustc_error_codes/src/error_codes/E0546.md

```diff
@@ -0,0 +1,11 @@
+A feature name is missing.
+
+Erroneous code example:
+
+```compile_fail,E0546
```

Comment:
[CI fails](https://github.com/rust-lang/rust/pull/79533/checks?check_run_id=1469768600) because this emit [E0734](https://doc.rust-lang.org/error-index.html#E0734).
How to fix that? This error seems mainly for rustc developers and is it good to remove example?

Reply:
Keep the example but instead use `ignore (error only exists when building std)`.